Add meta descriptions to blog posts that have not been updated (#1542)
pulumi/docs: content/blog/reusable-cicd-components-with-circleci-orbs-for-pulumi/index.md

authors: ["chris-smith"]
tags: ["CI/CD"]
date: "2018-11-07"

> meta_desc: "Pulumi is proud to be a CircleCI technology partner and we are continuing to make Pulumi the best tool for the continuous deployment part of CI/CD workflows. This blog post showcases Pulumi orbs in a CircleCI configuration using a simple job to build and update a JavaScript-based stack."

Reviewer: ```suggestion
meta_desc: "Pulumi is proud to be a CircleCI technology partner as we continue to make Pulumi the best tool for CI/CD workflows. This blog post showcases Pulumi Orbs with CircleCI, using a simple job to build and update a JavaScript-based stack."
```
